test/server/lib/payments.test.js.snap

```diff
@@ -2,24 +2,24 @@
-| CONTRIBUTION            | CREDIT | false    |        | User            | Collective      | Host            | 4500   | USD      | 0                         | -175       |            | Monthly subscription to Webpack                        |
-| PLATFORM_TIP            | DEBIT  | true     | false  | User            | Open Collective | Open Collective | -500   | USD      | 0                         | 0          |            | Refund of \\"Financial contribution to Open Collective\\"  |
-| PLATFORM_TIP            | CREDIT | true     | false  | Open Collective | User            | NULL            | 500    | USD      | 0                         | 0          |            | Refund of \\"Financial contribution to Open Collective\\"  |
-| PLATFORM_TIP_DEBT       | DEBIT  | true     | true   | Open Collective | Host            | Host            | -500   | USD      | 0                         | 0          |            | Refund of \\"Platform Tip collected for Open Collective\\" |
-| PLATFORM_TIP_DEBT       | CREDIT | true     | true   | Host            | Open Collective | Open Collective | 500    | USD      | 0                         | 0          |            | Refund of \\"Platform Tip collected for Open Collective\\" |
-| PAYMENT_PROCESSOR_COVER | DEBIT  | true     |        | Collective      | Host            | Host            | -175   | USD      | 0                         | 0          |            | Cover of payment processor fee for refund              |
-| PAYMENT_PROCESSOR_COVER | CREDIT | true     |        | Host            | Collective      | Host            | 175    | USD      | 0                         | 0          |            | Cover of payment processor fee for refund              |
-| HOST_FEE                | DEBIT  | true     |        | Collective      | Host            | Host            | -250   | USD      | 0                         | 0          |            | Refund of \\"Host Fee\\"                                   |
-| HOST_FEE                | CREDIT | true     |        | Host            | Collective      | Host            | 250    | USD      | 0                         | 0          |            | Refund of \\"Host Fee\\"                                   |
-| CONTRIBUTION            | DEBIT  | true     |        | User            | Collective      | Host            | -4500  | USD      | 0                         | 0          |            | Refund of \\"Monthly subscription to Webpack\\"            |
-| CONTRIBUTION            | CREDIT | true     |        | Collective      | User            | NULL            | 4500   | USD      | 0                         | 0          |            | Refund of \\"Monthly subscription to Webpack\\"            |"
+| kind                    | type   | isRefund | isDebt | From            | To              | Host            | amount | currency | platformFee | paymentFee | Settlement | description                                            |
```

Comment:
Huge diff there but it's only because of the new alias `platformFeeInHostCurrency` -> `platformFee`